challenges/2021-04-06-intervening-verbiage/challenge.md

```diff
@@ -23,7 +23,13 @@ count_words_between("an", "outhouse", list_of_words)    // 6
 
 ## Tackling This Challenge
 
-To complete this challenge, add your code to the 'mob.java' file. Confirm your solution by running tests by executing the `run_tests.sh` script (use `./run_tests.sh` from the challenge root directory.
+1. Clone (or pull if you've cloned already) the codeconnector/CodeingDojo repository using `git clone https://github.com/codeconnector/CodingDojo.git` or `git pull`, as appropriate. 
```

Comment:
We were having people besides us `fork` the repo, correct?

Otherwise, nice job.

Reply:
Makes sense, I'll update